app/models/profile.rb

```diff
@@ -13,7 +13,7 @@ class Profile < ApplicationRecord
   validate :iso_languages_array_has_right_format
   before_save :clean_iso_languages!
 
-  translates :bio, :main_topic, :twitter, :website, :website_2, :website_3, :city, fallbacks_for_empty_translations: true
+  translates :bio, :main_topic, :profession, :twitter, :website, :website_2, :website_3, :city, fallbacks_for_empty_translations: true
   accepts_nested_attributes_for :translations
   globalize_accessors locales: %i[en de], attributes: %i[main_topic bio twitter website website_2 website_3 city]
```

Comment:
if you add `:profession` here, we can access the attributes like that `profession_en` and `profession_de`